src/qt/paymentrequestplus.h

```diff
@@ -27,7 +27,6 @@ class PaymentRequestPlus
     bool SerializeToString(std::string* output) const;
 
     bool IsInitialized() const;
-    QString getPKIType() const;
```

Comment:
This method is unused. Is there a plan to make use of it soon? IMO we should keep unused implementation in master at minimum. Better keep this in a separate branch and add it if we use it.